README.md

```diff
@@ -125,15 +125,19 @@ Your config folder will be mounted in `/tmp/docker-mailserver/`. To understand h
 `restart: always` ensures that the mail server container (and Filebeat/ELK containers when using the mail server together with ELK stack) is automatically restarted by Docker in cases like a Docker service or host restart or container exit.
 
 #### Exposed ports
-* 25 receiving email from other mailservers
-* 465 SSL Client email submission
-* 587 TLS Client email submission
-* 143 StartTLS IMAP client
-* 993 TLS/SSL IMAP client
-* 110 POP3 client
-* 995 TLS/SSL POP3 client
-
-`Note: Port 25 is only for receiving email from other mailservers and not for submitting email. You need to use port 465 or 587 for this.`
+
+| Protocol | Opt-in Encryption<sup>1</sup> | Enforced Encryption | Purpose              |
+|----------|-------------------------------|---------------------|----------------------|
+| SMTP     | 25                            | N/A                 | Transfer<sup>2</sup> |
+| ESMTP    | 587                           | 465<sup>3</sup>     | Submission           |
+| POP3     | 110                           | 995                 | Retrieval            |
+| IMAP4    | 143                           | 993                 | Retrieval            |
```

Comment:
I suggest to clarify this too:

```suggestion
| POP3     | 110                           | 995                 | Retrieval (unsynced)           |
| IMAP4    | 143                           | 993                 | Retrieval (synced)           |
```

Reply:
I'm not sure if that's suitable here? If the user doesn't know the difference between POP and IMAP protocols, they're either going to support both or need to read up about them. "Retrieval" keeps it simple for the purpose served.

Wiki section should touch on setup that if using the image to retrieve from an inbox is desirable, that IMAP is most common these days and preferred. 

I don't know if we need to document the behaviour of these too much, other than the ports and what is considered secure. I trust that if the user is deploying this, they're capable of reading up about the basics of POP/IMAP, or already aware of it. 

Some users, will not want to support retrieval and just use the image for sending or relaying transactional email (albeit this image is a bit overkill for such, it has a high popularity / contributions and generally trusted more to be properly configured / secure and documented). These users just need to be aware of what ports/protocols matter for them, they're possibly not familiar with MTA/MSA but would know that retrieval isn't important.

Reply:
OK, just was a suggestion.